pages/home.html

```diff
@@ -21,20 +27,18 @@
     Bite-sized
   </h3>
   <p>
-    Explorations are easy to create and quick to play. They can be published in
-    the exploration gallery, shared on social media, or embedded on your
-    webpage.
+    Easy to create and quick to play, explorations can also be published in the exploration gallery, shared on social media, or embedded on your webpage.
```

Comment:
Sounds good! You also mentioned bringing it to the oppia-chat channel -- yep, feel free to do that, that's totally fine!